Microsoft.NET.Build.Containers/Exceptions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Runtime.Serialization;
+
+namespace Microsoft.NET.Build.Containers;
+
+public class DockerLoadException : Exception
+{
+    public DockerLoadException()
+    {
+    }
+
+    public DockerLoadException(string? message) : base(message)
+    {
+    }
+
+    public DockerLoadException(string? message, Exception? innerException) : base(message, innerException)
+    {
+    }
+
+    protected DockerLoadException(SerializationInfo info, StreamingContext context) : base(info, context)
+    {
+    }
+}
+
+public class ContainerHttpException : Exception
+{
+    private const string errorPrefix = "Containerize: error CONTAINER004:";
+    string? jsonResponse;
```

Comment:
It's questionable whether we need these as locals. I was just considering the debugging experience here.

Reply:
No objection to keeping the string around for potentially a bit longer in an error case if it helps debugging even a little bit.